views/default/forms/admin/menu/save.php

```diff
@@ -7,11 +7,7 @@
 $num_featured_items = 6;
 
 // get site menu items
-$menu = elgg_get_config('menus');
-$menu = $menu['site'];
-$builder = new ElggMenuBuilder($menu);
-$menu = $builder->getMenu('name');
-$menu_items = $menu['default'];
+$menu_items = elgg_get_menu('site')->getMenuItems('name', 'default');
```

Comment:
@cash Do you happen to remember why this didn't fire menu hooks? We're the hooks just not around yet? I see it was added in https://github.com/Elgg/Elgg/commit/8ec093ecc1bf28c09f5d2dbe3c7bc8e6a2485ac6#diff-3ee5cdf455fbc6d83500c13643124b0eR9